oracle_fdw.c

```diff
@@ -2774,15 +2779,15 @@ struct OracleFdwState
 	else
 		fdwState->prefetch = (unsigned int)strtoul(fetch, NULL, 0);
 
-	/* the limit for "prefetch" used to be higher than 1000 */
-	if (fdwState->prefetch > 1000)
+	/* Reduce "prefetch" if required to meet configured limit */
```

Comment:
It is good that you change the comment, but please retain the information that we need this bit of code because there may be tables out there from previous versions (where `prefetch` had a different meaning) with higher values set.